src/keri/app/cli/commands/escrow/clear.py

```diff
@@ -0,0 +1,57 @@
+# -*- encoding: utf-8 -*-
+"""
+KERI
+keri.kli.commands.escrow module
+
+"""
+import argparse
+
+from hio import help
+from hio.base import doing
+from keri.app.cli.common import existing
+from keri.vdr import viring
+
+logger = help.ogler.getLogger()
+
+parser = argparse.ArgumentParser(description='Clear escrows')
+parser.set_defaults(handler=lambda args: handler(args),
+                    transferable=True)
+parser.add_argument('--name', '-n', help='keystore name and file location of KERI keystore', required=True)
+parser.add_argument('--base', '-b', help='additional optional prefix to file location of KERI keystore',
+                    required=False, default="")
+parser.add_argument('--passcode', '-p', help='21 character encryption passcode for keystore (is not saved)',
```

Comment:
Do you need `--nopasscode` here too in order to match the other commands?

Reply:
As far as I can tell then `--nopasscode` is only needed in the `kli init` command because that is where the Habery, keeping.Manager are set up based on whether a passcode is supplied or not. If `--nopasscode` is set when `kli init` is called then you don't need to provide a passcode on later commands, so it is okay that `kli escrow clear` and `kli escrow list` do not have this argument.

Reply:
Oh cool.